pyseir/inference/infer_rt.py

```diff
@@ -361,7 +361,14 @@ def get_posteriors(self, timeseries_type, plot=False):
             denominator = np.sum(numerator)
 
             # Execute full Bayes' Rule
-            posteriors[current_day] = numerator / denominator
+            if denominator == 0:
+                # restart the baysian learning for the remaining series
+                # this is necessary since otherwise NaN values
+                # will be inferred for all future days, after seeing
+                # a single (smoothed) zero value.
+                posteriors[current_day] = prior0
```

Comment:
Can you add another note that captures 'We understand that restarting the posteriors with the initial prior may incur a start-up artifact as the posterior restabilizes, but that we believe it's the current best solution for edge cases like X.' Let's give future selves and others more hints into the quirks of the system.

Reply:
But I guess this implies we aren't going to test artificially increasing the denominator to say 1E-10?